### PR TITLE
Efrs 1047 create image collections component

### PR DESCRIPTION
**Pull request by smchedlidze826**
EFRS-1047 and EFRS-1168  tasks are combined in this commit

### CODE REVIEW

**Comment by smchedlidze826** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by isobolev** [comment]
please remove node_modules folder

File: ui/src/app/core/collection/collection.service.ts [hunk 1]
```diff
@@ -55,12 +55,26 @@ export class CollectionService {
...
-      .get(`${environment.userApiUrl}recognition/faces?size=1000`, {
+      .get(`${environment.userApiUrl}recognition/faces?size=${size}&subject=${subject}&page=${page}`, {
         headers: { 'x-api-key': apiKey },
       })
-      .pipe(map((resp: { faces: SubjectExampleResponseItem[] }) => resp.faces));
+      .pipe(
+        map((resp: { faces: SubjectExampleResponseItem[] }) => {
+          const totalPages = resp['total_pages'];
+          const transformedArr = [];
+          resp.faces.map(el => {
```
Comment (isobolev) [inline ui/src/app/core/collection/collection.service.ts:72]: transformedArr is redundant; just return result of the resp.faces.map

```
return resp.faces.map(el => ({ ...el, page: page, totalPages: totalPages }));
```

File: ui/src/app/features/drag-n-drop/drag-n-drop.component.ts [hunk 2]
```diff
@@ -24,6 +24,7 @@ export class DragNDropComponent {
   @ViewChild('fileDropRef') fileDropEl: ElementRef;
   @Input() title: string;
   @Input() label: string;
+  size: string = '5Mb (5242880bytes)';
```
Comment (isobolev) [inline ui/src/app/features/drag-n-drop/drag-n-drop.component.ts:27]: should come from en.json

File: ui/src/app/store/manage-collectiom/reducers.ts [hunk 3]
```diff
@@ -113,21 +115,51 @@ const reducer: ActionReducer<CollectionEntityState> = createReducer(
...
     const newCollectionItems = items.map(item => ({
       url: CollectionHelper.getCollectionItemUrl(apiKey, item.image_id),
       id: item.image_id,
       status: CircleLoadingProgressEnum.Uploaded,
       subject: item.subject,
+      page: item['page'],
+      totalPages: item['totalPages'],
     }));
 
+    console.log(newCollectionItems);
```
Comment (isobolev) [inline ui/src/app/store/manage-collectiom/reducers.ts:131]: please remove console.log

Comment (smchedlidze826) [reply ui/src/app/store/manage-collectiom/reducers.ts:131]: sure, done

File: ui/src/app/features/drag-n-drop/drag-n-drop.component.ts (hunk 2 above)
Comment (smchedlidze826) [reply ui/src/app/features/drag-n-drop/drag-n-drop.component.ts:27]: Fixed

**Pull request merged**